Update az.m3u

Emilselimli opened the pull request:
**added new azerbaijan channels 
added : sahiyya tv {geoblocked) (unchanging token)
added : mtv (geoblocked) (unchanging token)**

Emilselimli committed:
Update az.m3u

added new azerbaijan channels 
added : stv (unchanging token)

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -45,3 +45,11 @@ https://cdn10-mugantv.yayin.com.tr/mugantv/mugantv/playlist.m3u8
+#EXTINF:9 tvg-id="AgroTV.tr" tvg-logo="https://i.imgur.com/ls9NLa4.png",AGRO TV AZERBAIJAN
+https://yayin30.haber100.com/live/agrotv2/chunklist_w27305036.m3u8
+#EXTINF:9 tvg-id="" tvg-logo="https://i.imgur.com/ls9NLa4.png", STV 
+http://api.canlitvplayer.com/str.yodacdn.net/shtv/mono.m3u8?token=49b46a963601772e25ced2b2886605de86b3c411-4d908a3ca4eb8248cbbcafc51e12d454-1664271581-1664260781
```
```suggestion
https://str.yodacdn.net/shtv/playlist.m3u8
```
?

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -45,3 +45,11 @@ https://cdn10-mugantv.yayin.com.tr/mugantv/mugantv/playlist.m3u8
 https://str1.yodacdn.net/qafkaz/playlist.m3u8
 #EXTINF:-1 tvg-id="RealTV.az",Real TV [Geo-blocked]
 https://str.yodacdn.net/real/playlist.m3u8
+#EXTINF:8  tvg-id="AlZahraTV.ir"tvg-logo="https://i.imgur.com/KsZoXBR.png",ZAHRA TV
```
```suggestion
#EXTINF:-1 tvg-id="AlZahraTV.ir", AL-ZAHRA TV
```

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -45,3 +45,11 @@ https://cdn10-mugantv.yayin.com.tr/mugantv/mugantv/playlist.m3u8
 https://str.yodacdn.net/real/playlist.m3u8
+#EXTINF:8  tvg-id="AlZahraTV.ir"tvg-logo="https://i.imgur.com/KsZoXBR.png",ZAHRA TV
+https://mn-nl.mncdn.com/alzahra/alzahra_hd.smil/playlist.m3u8
+#EXTINF:9 tvg-id="AgroTV.tr" tvg-logo="https://i.imgur.com/ls9NLa4.png",AGRO TV AZERBAIJAN
```
```suggestion
#EXTINF:-1 tvg-id="AgroTV.tr",AGRO TV AZERBAIJAN
```

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -45,3 +45,11 @@ https://cdn10-mugantv.yayin.com.tr/mugantv/mugantv/playlist.m3u8
+#EXTINF:8  tvg-id="AlZahraTV.ir"tvg-logo="https://i.imgur.com/KsZoXBR.png",ZAHRA TV
+https://mn-nl.mncdn.com/alzahra/alzahra_hd.smil/playlist.m3u8
+#EXTINF:9 tvg-id="AgroTV.tr" tvg-logo="https://i.imgur.com/ls9NLa4.png",AGRO TV AZERBAIJAN
+https://yayin30.haber100.com/live/agrotv2/chunklist_w27305036.m3u8
```
```suggestion
https://yayin30.haber100.com/live/agrotv2/playlist.m3u8
```

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -45,3 +45,11 @@ https://cdn10-mugantv.yayin.com.tr/mugantv/mugantv/playlist.m3u8
+https://yayin30.haber100.com/live/agrotv2/chunklist_w27305036.m3u8
+#EXTINF:9 tvg-id="" tvg-logo="https://i.imgur.com/ls9NLa4.png", STV 
+http://api.canlitvplayer.com/str.yodacdn.net/shtv/mono.m3u8?token=49b46a963601772e25ced2b2886605de86b3c411-4d908a3ca4eb8248cbbcafc51e12d454-1664271581-1664260781
+#EXTINF:1 tvg-logo="https://yoda.az/logos/mtv.png" 	tvg-id="MuzTV.az", MUZ TV
```
```suggestion
#EXTINF:-1 tvg-id="MuzTV.az", MUZ TV
```

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -45,3 +45,11 @@ https://cdn10-mugantv.yayin.com.tr/mugantv/mugantv/playlist.m3u8
+#EXTINF:9 tvg-id="" tvg-logo="https://i.imgur.com/ls9NLa4.png", STV 
+http://api.canlitvplayer.com/str.yodacdn.net/shtv/mono.m3u8?token=49b46a963601772e25ced2b2886605de86b3c411-4d908a3ca4eb8248cbbcafc51e12d454-1664271581-1664260781
+#EXTINF:1 tvg-logo="https://yoda.az/logos/mtv.png" 	tvg-id="MuzTV.az", MUZ TV
+https://str.yodacdn.net/mtvaz/playlist.m3u8
```
```suggestion
http://api.canlitvplayer.com/str.yodacdn.net/mtvaz/playlist.m3u8?token=49b46a963601772e25ced2b2886605de86b3c411-4d908a3ca4eb8248cbbcafc51e12d454-1664271581-1664260781
```

Emilselimli committed:
Update az.m3u

BellezaEmporium the user showed errors were corrected
and ARB Gunesh TV iplink changed (token does not change)

Emilselimli committed:
Update az.m3u

Emilselimli committed:
Update az.m3u

muztv replace link

Emilselimli committed:
Update az.m3u

Added Channels 
CBC SPORT
CBC (replaced broken link)
Xezer TV

(token do not chnaged)

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -21,27 +29,31 @@ http://85.132.81.184:8080/atv-4/index.m3u8
 https://str1.yodacdn.net/qafkaz/playlist.m3u8
 #EXTINF:-1 tvg-id="RealTV.az",Real TV [Geo-blocked]
 https://str.yodacdn.net/real/playlist.m3u8
+#EXTINF:-1 tvg-id="AlZahraTV.ir", AL-ZAHRA TV
```
```suggestion
#EXTINF:-1 tvg-id="AlZahraTV.ir",AL-ZAHRA TV
```

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -21,27 +29,31 @@ http://85.132.81.184:8080/atv-4/index.m3u8
+https://mn-nl.mncdn.com/alzahra/alzahra_hd.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AgroTV.tr",AGRO TV AZERBAIJAN [Geo-blocked]
+https://yayin30.haber100.com/live/agrotv2/playlist.m3u8
+#EXTINF:-1 tvg-id="", Sahiyya TV [Geo-blocked]
```
```suggestion
#EXTINF:-1 tvg-id="",Sahiyya TV [Geo-blocked]
```

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -21,27 +29,31 @@ http://85.132.81.184:8080/atv-4/index.m3u8
+https://yayin30.haber100.com/live/agrotv2/playlist.m3u8
+#EXTINF:-1 tvg-id="", Sahiyya TV [Geo-blocked]
+http://api.canlitvplayer.com/str.yodacdn.net/shtv/mono.m3u8?token=49b46a963601772e25ced2b2886605de86b3c411-4d908a3ca4eb8248cbbcafc51e12d454-1664271581-1664260781
+#EXTINF:-1 tvg-id="MuzTV.az", MUZ TV [Geo-blocked]
```
```suggestion
#EXTINF:-1 tvg-id="MuzTV.az",MUZ TV [Geo-blocked]
```

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -21,27 +29,31 @@ http://85.132.81.184:8080/atv-4/index.m3u8
+http://api.canlitvplayer.com/str.yodacdn.net/shtv/mono.m3u8?token=49b46a963601772e25ced2b2886605de86b3c411-4d908a3ca4eb8248cbbcafc51e12d454-1664271581-1664260781
+#EXTINF:-1 tvg-id="MuzTV.az", MUZ TV [Geo-blocked]
+http://api.canlitvplayer.com/str.yodacdn.net/mtvaz/mono.m3u8?token=52078ae03437564a6f7037aed547f047c0466f3e-d3d765684657c8fc3a0cafb695d79af3-1664271354-1664260554
+#EXTINF:-1 tvg-id="XezerTV.az", Xezer TV [Geo-blocked]
```
```suggestion
#EXTINF:-1 tvg-id="XezerTV.az",Xezer TV [Geo-blocked]
```

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -21,27 +29,31 @@ http://85.132.81.184:8080/atv-4/index.m3u8
 https://rtmp.baku.tv/live/bakutv_720p.m3u8
-#EXTINF:-1 tvg-id="CBC.az",CBC (720p)
-http://cbctvlive.flashmediacast.com:1935/CBCTVLive/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="CBC.az",CBC (720p)  [Geo-blocked]
```
```suggestion
#EXTINF:-1 tvg-id="CBC.az",CBC (720p) [Geo-blocked]
```

Emilselimli committed:
Update az.m3u

update playlist

Emilselimli committed:
Update az.m3u

update

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -21,27 +29,31 @@ http://85.132.81.184:8080/atv-4/index.m3u8
 https://str1.yodacdn.net/qafkaz/playlist.m3u8
 #EXTINF:-1 tvg-id="RealTV.az",Real TV [Geo-blocked]
 https://str.yodacdn.net/real/playlist.m3u8
+#EXTINF:-1 tvg-id="AlZahraTV.ir",AL-ZAHRA TV
```
To be moved to the "ir.m3u" playlist, to correspond with the tvg-id.

BellezaEmporium replied:
Indeed, but we classify the channel by their country of origin, not by the intended countries they broadcast to.

It is a channel based in Iran, so it should be classified in the ir.m3u playlist. It would still appear on the Azerbaijan playlist (after CI publication) as it is made for their diaspora.

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -21,27 +29,31 @@ http://85.132.81.184:8080/atv-4/index.m3u8
 https://str.yodacdn.net/real/playlist.m3u8
+#EXTINF:-1 tvg-id="AlZahraTV.ir",AL-ZAHRA TV
+https://mn-nl.mncdn.com/alzahra/alzahra_hd.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AgroTV.tr",AGRO TV AZERBAIJAN [Geo-blocked]
```
To move with the "tr.m3u" playlist, to correspond with the tvg-id.

BellezaEmporium replied:
Indeed, but we classify the channel by their country of origin, not by the intended countries they broadcast to.

It is a channel based in Turkey, so it should be classified in the tr.m3u playlist. It would still appear on the Azerbaijan playlist (after CI publication) as it is made for their diaspora.

BellezaEmporium commented on streams/az.m3u:
```diff
@@ -21,27 +29,31 @@ http://85.132.81.184:8080/atv-4/index.m3u8
+https://mn-nl.mncdn.com/alzahra/alzahra_hd.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AgroTV.tr",AGRO TV AZERBAIJAN [Geo-blocked]
+https://yayin30.haber100.com/live/agrotv2/playlist.m3u8
+#EXTINF:-1 tvg-id="",Sahiyya TV [Geo-blocked]
```
other name https://sehiyyetv.az/

needs to be added into the database before merging.

BellezaEmporium replied:
You can create an issue on @iptv-org/database to add this channel in. Or we keep it as-is until we add the necessary data on it.

Emilselimli committed:
Update az.m3u

update azerbaijan playlist

Emilselimli reviewed (commented):
UPDATE PLAYLIST

freearhey commented:
Closed due to lack of activity.
